python/ray/autoscaler/_private/load_metrics.py

```diff
@@ -23,6 +22,47 @@
 ])
 
 
+def add_resources(dict1: Dict[str, float],
```

Comment:
move from util to here to solve circular dependency when using the LoadMetricsSummary type